CONTRIBUTING.md

```diff
@@ -52,11 +52,15 @@ You can do this online, and it only takes a minute.
 **Before setting up your DevTools environment**, please make sure you have
 [cloned the Flutter SDK from GitHub](https://github.com/flutter/flutter/blob/main/docs/contributing/Setting-up-the-Framework-development-environment.md)
 and added the included `flutter` and `dart` executables to your `PATH` environment variable (see Flutter
-instructions for how to [update your PATH](https://flutter.dev/to/update-macos-path)).
+instructions for how to [update your PATH](https://docs.flutter.dev/get-started/install/macos/mobile-ios#add-flutter-to-your-path)).
```

Comment:
Can you revert this change? I'm updating the website now so the `/to/` redirect works. While the specific path and fragment are likely to break.

```suggestion
instructions for how to [update your PATH](https://flutter.dev/to/update-macos-path)).
```